src/scheduler.rs

```diff
@@ -140,6 +173,66 @@ async fn scheduler_loop(
 
         debug!(tick = %tick, "Scheduler ticking");
         schedule_checks(tick);
+        let _: () = connection
```

Comment:
Why did you have to type this?
```suggestion
        let _ = connection
```

Reply:
Rust complains otherwise 🤷

Reply:
Actually also mentioned in the docs for redis-rs https://github.com/redis-rs/redis-rs